### PR TITLE
add note re: prequisite packages with pip install

### PR DESCRIPTION
**Pull request by dlo**
**Description:**

See #8372 

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_faq/pip3.markdown [hunk 1]
```diff
@@ -2,7 +2,7 @@
 layout: page
 title: "pip3: command not found"
 description: "pip3: command not found"
-date: 2017-06-18 09:00
+date: 2019-02-01 09:57
```
Comment (klaasnicolaas) [inline source/_faq/pip3.markdown]: Please put this back

Comment (dlo) [reply source/_faq/pip3.markdown]: Just reverted. Thanks.

**Review by frenck — APPROVED** [review]
Thanks, @dlo 👍

**Comment by dlo** [comment]
🕺🕺🕺

**Pull request merged**